app/views/shell/_top.html.erb

```diff
@@ -11,21 +11,20 @@
       <meta name="algolia-public-key" content="<%= ALGOLIASEARCH_PUBLIC_SEARCH_ONLY_KEY %>">
       <meta name="environment" content="<%= Rails.env %>">
       <%= render "layouts/styles" %>
-      <style> 
-        .home { 
-          overflow: hidden; 
-          position: relative; 
-          margin: auto; 
-          max-width: 1250px;  
-        } 
+      <style>
+        .home {
+          overflow: hidden;
+          position: relative;
+          margin: auto;
+          max-width: 1250px;
+        }
       </style>
-      <%= javascript_pack_tag "manifest", defer: true %>
-      <%= javascript_pack_tag "vendor", defer: true %>
```

Comment:
don't we need the other two? are they implicitly required?

Reply:
Same as above. 😄